### ISSUE TITLE
[ordered-imports] `"import-sources-order": any` invalidate "grouped-imports"

### ISSUE THREAD

**Issue by Kerumen** [palantir/tslint#4338]
### Bug Report

- __TSLint version__: `v5.11.0`
- __TypeScript version__: `v3.1.6`
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
import { getOr } from 'lodash/fp'

import { Request } from 'express'
import { getStatusCode } from './errors'
```

with `tslint.json` configuration:

```json
{
  "defaultSeverity": "warning",
  "rules": {
    "ordered-imports": [
      true,
      {
        "grouped-imports": true,
        "import-sources-order": "any"
      }
    ]
  }
}
```

#### Actual behavior

The `import { Request } from 'express'` is **not** reported with:
_"Import sources of different groups must be sorted by: libraries, parent directories, current directory."_

#### Expected behavior

If I change `"import-sources-order"` to any value but `any` or if I remove it from the rule config, I get _"Import sources of different groups must be sorted by: libraries, parent directories, current directory."_ which is **good**.

The `any` parameter should not invalidate the `grouped-imports` config.

PS: We have the same behaviour with the other config, `named-imports-order`.
